scripts/auto-cite.py

```diff
@@ -0,0 +1,67 @@
+# auto-cite core runner
+
+# packages
+from util import *
+
+# auto-cite plugins
+from orcid import main as orcid
+from sources import main as sources
+
+# filename for output citations
+filename = "../_data/citations.yaml"
+
+# plugin order
+plugins = [orcid, sources]
+
+section()
+
+info("Compiling sources")
+
+# compile input sources from various plugins
+sources = []
```

Comment:
I like this approach! It looks like you could make the plugins that get run configurable, which seems like a huge plus. Folks could pick one or many to setup.

Reply:
It's hard to say what kind of configuration would be required before looking at these systems. Right now I can only imagining needing to pass a URL, perhaps something like:

```
plugins = [orcid, university("https://api.cu.edu/students/$ID"), sources]
```